coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Your log files and coverage should not be showing up here. Check your `.gitignore` file